server/.ebextensions/resources.config

```diff
@@ -0,0 +1,5 @@
+Resources:
+  AWSEBAutoScalingGroup:
+    Type: AWS::AutoScaling::AutoScalingGroup
+    Properties:
+      PlacementGroup: "Instant-docker-prod-placement-group"
```

Comment:
I figured out how to make elasticbeanstalk put our instances in a placement group!

Reply:
Daam!